src/libxtp/jobcalculators/iqm.cc

```diff
@@ -302,8 +302,9 @@ Job::JobResult IQM::EvalJob(const Topology& top, Job& job, QMThread& opThread) {
     dft_logger.setPreface(Log::debug, (format("\nDFT DBG ...")).str());
     std::string dftname = "package.name";
     std::string package = _dftpackage_options.get(dftname).as<std::string>();
-    std::unique_ptr<QMPackage> qmpackage = std::unique_ptr<QMPackage>(
-        QMPackageFactory::QMPackages().Create(package));
+    QMPackageFactory factory;
+    std::unique_ptr<QMPackage> qmpackage =
+        std::unique_ptr<QMPackage>(factory.Create(package));
```

Comment:
you do not have to wrap the factory call in a unique_ptr anymore, because the factory returns a unique_ptr now.

Reply:
thanks for let me know!